src/components/molecules/molecules-readme.md

```diff
@@ -0,0 +1,5 @@
+# Patterns to follow while creating molecules
+- Composed of other molecules or atoms.
+- Use CSS to style .
+- Dumb components avoid using logic at this level.
```

Comment:
avoid using business logic
may be we can use `generic validations`

Reply:
yes validation is valid...will include